vendor/vendor.json

```diff
@@ -0,0 +1,268 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "E2a5TaCbwqs7PwP1fvfWLhvgr38=",
+			"comment": "v2.2.0-rc.0-72-g106d918",
+			"path": "github.com/coreos/etcd/Godeps/_workspace/src/github.com/coreos/go-systemd/journal",
```

Comment:
Cloning both repositories:
```
$ git clone git@github.com:mesosphere/etcd-mesos.git
$ git clone https://github.com/coreos/etcd.git
$ cd etcd && git checkout 106d918dd585a63ee0045740fe040daa44669a77
```

Diff'ing resources that exist in both directories simultaneously:
```
$ diff etcd-mesos/Godeps/_workspace/src/github.com/coreos/etcd/etcdserver/stats/ etcd/etcdserver/stats/
diff etcd-mesos/Godeps/_workspace/src/github.com/coreos/etcd/etcdserver/stats/stats.go etcd/etcdserver/stats/stats.go
17c17
< import "github.com/coreos/pkg/capnslog"
---
> import "github.com/coreos/etcd/Godeps/_workspace/src/github.com/coreos/pkg/capnslog"
```
```
$ diff etcd-mesos/Godeps/_workspace/src/github.com/coreos/etcd/etcdserver/etcdhttp/httptypes/ etcd/etcdserver/etcdhttp/httptypes/
diff etcd-mesos/Godeps/_workspace/src/github.com/coreos/etcd/etcdserver/etcdhttp/httptypes/errors.go etcd/etcdserver/etcdhttp/httptypes/errors.go
21c21
< 	"github.com/coreos/pkg/capnslog"
---
> 	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/coreos/pkg/capnslog"
```

Now, in `Godeps/Godeps.json` we have: 
```json
		{
			"ImportPath": "github.com/coreos/go-systemd/journal",
			"Comment": "v3-11-g6f3d6b0",
			"Rev": "6f3d6b0accd17b33624ee4df1532a03de6e4d536"
		},
```
But in the official code we actually have https://github.com/coreos/etcd/blob/106d918dd585a63ee0045740fe040daa44669a77/Godeps/_workspace/src/github.com/coreos/pkg/capnslog/journald_formatter.go#L24.

I conclude there is no code that has been changed other than imports and we should clean this up when adopting etcd v3.x.